roles/install_kubectl_versions/tasks/process-install.yaml

```diff
@@ -15,25 +15,62 @@
     - avinode_devenv_install_kubectl
 
 - name: Fetech kubectl checksum ({{ version }})
-  uri:
-    return_content: yes
-    url: https://dl.k8s.io/v{{ version }}/bin/{{ go_platform }}/{{ go_arch }}/kubectl.sha256
-  register: _kubectl_checksum
-  tags:
-    - avinode_devenv_install_kubectl
+  block:
+    - name: Fetech kubectl checksum ({{ version }} arch {{ go_arch }})
+      uri:
+        return_content: yes
+        url: https://dl.k8s.io/v{{ version }}/bin/{{ go_platform }}/{{ go_arch }}/kubectl.sha256
+      register: _kubectl_checksum
+      tags:
+        - avinode_devenv_install_kubectl
+  rescue:
+    - block:
+      - name: Check if fallback
+        fail:
+          msg: "Could not find file and no fallback set."
+        when: go_arch_fallback is not defined
+      - name: Fetech kubectl checksum ({{ version }} fallback arch)
+        uri:
+          return_content: yes
+          url: https://dl.k8s.io/v{{ version }}/bin/{{ go_platform }}/{{ go_arch_fallback }}/kubectl.sha256
+        register: _kubectl_checksum
+        tags:
+          - avinode_devenv_install_kubectl
```

Comment:
I'm wondering if we could use something like this:
- https://medium.com/opsops/how-to-break-from-the-loop-in-ansible-1e8ebb92be0d